lib/hq/graphql/input_object.rb

```diff
@@ -32,19 +32,19 @@ def format_nested_attributes
       end
 
       #### Class Methods ####
-      def self.with_model(model_name, attributes: true, associations: false, enums: true)
+      def self.with_model(model_name, excluded_inputs = [], attributes: true, associations: false, enums: true)
```

Comment:
This should be a keyword argument
```suggestion
      def self.with_model(model_name, attributes: true, associations: false, enums: true, excluded_inputs: [])
```